### DISCUSSION TITLE
Can we add `workflow` label to our current labelling system

### DISCUSSION THREAD

**Discussion by mikkokotila** [world-federation-of-advertisers/cross-media-measurement#1909] [Ideas & Requests]
Right now, there is no label to assign for issues which focus on CI and other use of GitHub actions, or whatever improvements pertaining to the GitHub aspect. The closest label for this kinds of case would be `enhancement` which is described as a "New feature or request".

Would be good to add it, not sure if `workflow` is the ideal name though as it pertains just to actions in GitHub language, but it would be good of this label covers everything having to do with GitHub. Description could be something on the lines of:

**"An improvement or change specific to GitHub"**

**Comment by SanjayVas** [comment]
I think a `process` label might be better, which covers anything that's about process rather than code. This can include changes to CI/CD (usually done by GitHub Actions workflows).

I'm obviously a bit biased, but I default to referring to the [Google Issue Tracker documentation](https://developers.google.com/issue-tracker/concepts/issues#type) for how to do issue management. Note that some of the issue types there are specifically for the project management use case, and this project currently does not use GitHub issues for that purpose.

**Comment by mikkokotila** [comment]
I've added label `process` with the definition **"Anything that's about process rather than code"**.

Cliosing here.
